### DISCUSSION TITLE
Stratis for the home directory?

### DISCUSSION THREAD

**Discussion by fbruetting** [stratis-storage/project#379] [Ideas]
Hi! I just read through the RHEL 8 feature notes and read that snapshotting now is possible via Stratis – COOL! Sadly, I was not able to find documentation about if it’s already a good idea to use Stratis for the home directory file system – just that it cannot be used for the primary file system. Would you please consider documenting that topic?

If it’s not yet possible/recommended, it would be cool to outline the reasons in an FAQ entry. Otherwise it would be nice if you could supply information on how to use Stratis for a second home file system (In the Live System before running Anaconda?). I think this would be of a lot of interest, especially with a Silverblue OSTree/Stratis spilt 😁.

P.S.: Is it possible to back-up snapshots by deltas? I know you don’t have “Stratis send/receive”, but could that maybe be done by rsyncing? Don’t know if this is possible or a good idea… 🤔😬 Sorry if this is a silly question. 🙈

**Comment by mulkieran** [comment]
We have a lot more discussion of Stratis and filesystem management in our blog post about Stratis support for the root filesystem[1] which was introduced in a previous release.

I'm not sure I understand the question about backing-up snapshots by deltas.

[1] https://stratis-storage.github.io/stratis-rootfs/
